src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.ActiveDirectory.tests.ps1

```diff
@@ -0,0 +1,61 @@
+Import-Module Microsoft.Graph.Applications
+Import-Module -Name $PSScriptRoot\..\Arcus.Scripting.ActiveDirectory -ErrorAction Stop
+
+InModuleScope Arcus.Scripting.ActiveDirectory {
+    Describe "Arcus Azure Active Directory integration tests" {
+        BeforeEach {
+            $config = & $PSScriptRoot\Load-JsonAppsettings.ps1
+            & $PSScriptRoot\Connect-AzAccountFromConfig.ps1 -config $config
+        }
+        Context "Add an Active Directory Application Role Assignment" {
+            It "Creating a new role and assigning it should succeed" {
+                # Arrange
+                $ClientId = $config.Arcus.ActiveDirectory.MainAppClientId
+                $RoleName = 'SomeRole'
+                $AssignRoleToClientId = $config.Arcus.ActiveDirectory.ClientAppClientId
+
+                # Act
+                {
+                   Add-AzADAppRoleAssignment -ClientId $ClientId -Role $RoleName -AssignRoleToClientId $AssignRoleToClientId
+                } | Should -Not -Throw
+
+                # Assert
+                $adApplication = Get-AzADApplication -Filter "AppId eq '$ClientId'"
+                $RoleName | Should -BeIn $adApplication.AppRole.Value
+
+                $adServicePrincipal = Get-AzADServicePrincipal -Filter "AppId eq '$ClientId'"
+                $appRoleAssignments = Get-MgServicePrincipalAppRoleAssignedTo -ServicePrincipalId $adServicePrincipal.Id
+                $appRoleAssignments.Count | Should -Be 1
+            }
+            It "Get the role and assignment should succeed" {
+                # Arrange
+                $ClientId = $config.Arcus.ActiveDirectory.MainAppClientId
+                $RolesAssignedToClientId = $config.Arcus.ActiveDirectory.ClientAppClientId
+
+                # Act
+                {
+                   Get-AzADAppRoleAssignments -ClientId $ClientId -RolesAssignedToClientId $RolesAssignedToClientId
```

Comment:
Maybe we can add an assertion to verify that result here is a non-empty sequence?

Reply:
Ah, so if it does not return anything, maybe we can rename the script, as `Get...` makes me wonder to 'get' something 😄 .
Maybe `Verify-Az...` or `Has-Az...` and return a boolean or something?

Reply:
Good point, I think `List-AzADAppRoleAssignments` makes sense, agree?

Reply:
Ah yes, that's a good one. 👍

Reply:
Changed in [`d544b98` (#336)](https://github.com/arcus-azure/arcus.scripting/pull/336/commits/d544b98566a9735b0a979c87140d2719e7e664b4)